main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -173,7 +177,23 @@ DataTableView.prototype._renderPagingControls = function(pageSizeControls, pagin
     previousPage.addClass("inaction");
   }
 
-  $('<span>').addClass("viewpanel-pagingcount").html(" " + from + " - " + to + " ").appendTo(pagingControls);
+  lastPageSpan = $('<span>').attr("id", "viewpanel-paging-last").text(self._lastPageNumber);
+  pageControlSpan = $('<span>').attr("id", "viewpanel-paging-current");
+  
+  currentPageInput = $('<input type="number">')
+    .change(function(evt) { self._onChangeGotoPage(this, evt); })
+    .attr("id", "viewpanel-paging-current-input")
+    .attr("min", 1)
+    .attr("max", self._lastPageNumber)
+    .attr("id", "viewpanel-paging-current-input")
```

Comment:
I'd also add `required` so we don't accept an empty value
REF: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#Validation
```suggestion
    .attr("id", "viewpanel-paging-current-input")
    .attr("required")
```

Reply:
Why is "id" attr repeated?

Reply:
Dumb mistake. I do them sometimes… ;-)

Reply:
> I'd also add `required` so we don't accept an empty value
> REF: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#Validation

Not sure if the required will change something, because I was managing it, but I guess it can't hurt. Added.